### PR TITLE
* Add structures for SMSG_CHROMIE_TIME_OPEN_NPC, CMSG_CHROMIE_TIME

### PR DESCRIPTION
**Pull request by Mistixx**
structure for npc Chromie functionality new shadowowlands leveling system
chromie MistiX

### CODE REVIEW

File: WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using System;
+using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+using WowPacketParser.Proto;
+using WowPacketParser.Store;
+using WowPacketParser.Store.Objects;
+using CoreParsers = WowPacketParser.Parsing.Parsers;
```
Comment (MaxtorCoder) [inline WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs:8]: Remove unused using directives.

File: WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            packet.ReadPackedGuid128("GUID");
+        }
+
+        [Parser(Opcode.CMSG_CHROMIE_TIME_SELECT_EXPANSION)]
+        public static void HandleChromieTimeSelectExpansion(Packet packet)
+        {
+            packet.ReadPackedGuid128("GUID");
+            packet.ReadUInt32("Expansion");
+        }
+    }
```
Comment (MaxtorCoder) [inline WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs:27]: Just move these to NpcHandler.cs, there are only 3 packets for Chromie and it's NPC related.

**Review by MaxtorCoder — CHANGES_REQUESTED** [review]

File: WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs (hunk 2 above)
Comment (Mistixx) [reply WowPacketParserModule.V9_0_1_36216/Parsers/ChromieHandler.cs:27]: ok thank you

File: WowPacketParserModule.V9_0_1_36216/Parsers/NpcHandler.cs [hunk 3]
```diff
@@ -84,6 +84,19 @@ public static void HandleGossipPoi(Packet packet)
                     tempGossipOptionPOI.Reset();
                 }
             }
+        }
+		
+		[Parser(Opcode.SMSG_CHROMIE_TIME_OPEN_NPC)]
```
Comment (funjoker) [inline WowPacketParserModule.V9_0_1_36216/Parsers/NpcHandler.cs:89]: tabs

Comment (Mistixx) [reply WowPacketParserModule.V9_0_1_36216/Parsers/NpcHandler.cs]: ready

**Comment by funjoker** [comment]
thanks :)

**Pull request merged**